shared/src/packages/users/libs/types/user-entity.type.ts

```diff
@@ -25,8 +26,19 @@ type UserEntityObjectWithGroupAndBusinessT = UserEntityObjectWithGroupT & {
   business: BusinessEntityT;
 };
 
+type UserEntityObjectWithGroupAndDriverT = UserEntityObjectWithGroupT & {
+  driver: DriverWithAvatarUrl;
+};
+
+type FrontendAuthUser =
```

Comment:
the name is a bit confusing I think, it's named Frontend.., and used in shared, be, fe
let's name it to type AuthUser please